main.go

```diff
@@ -54,13 +54,21 @@ import (
 )
 
 var (
-	setupLog   = ctrl.Log.WithName("entrypoint")
-	logOptions = logs.NewOptions()
-
-	managerOpts     manager.Options
-	webhookOpts     webhook.Options
-	syncPeriod      time.Duration
-	profilerAddress string
+	setupLog       = ctrl.Log.WithName("entrypoint")
+	logOptions     = logs.NewOptions()
+	controllerName = "cluster-api-provider-vsphere-manager"
```

Comment:
```suggestion
	controllerName = "cluster-api-vsphere-manager"
```
maybe. Not sure if there is any consistency for that stuff